config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -99,6 +99,49 @@ presubmits:
       testgrid-dashboards: sig-windows-presubmit
       testgrid-tab-name: pull-kubernetes-e2e-capz-windows-serial-slow-hpa
       testgrid-num-columns-recent: '30'
+  - name: pull-kubernetes-e2e-capz-windows-cri-stats-alpha
```

Comment:
should we just name this something like `pull-kubernetes-e2e-capz-windows-alpha-features` and then evolve this to run different tests over time?